lesson6/README.md

```diff
@@ -0,0 +1,87 @@
+  * Access to important actions in a predictable way, such as search.
+  * Support for navigation and view switching (with tabs or drop-down lists).
+
+#### [Settings](https://developer.android.com/guide/topics/ui/settings.html)
+Remember how one of the design principles said "Let me make it mine"? We want to provide them with good defaults but always provide easy ways to change those defaults. Settings page just does that.
+
+If you want to learn about how to design a Settings page, [read this guide](https://material.google.com/patterns/settings.html#settings-usage).
+
+#### [Pop-up messages](https://developer.android.com/training/snackbar/index.html)
+Whenever you need to provide the users with a prominent enough message, use pop-up messages. Most of the time, the user doesn't have to respond to the message. Also, the message disappears after a short while. The pop-up message should not block the users from using the app.
+
+![Pop-up message with undo action](https://developer.android.com/images/training/snackbar/snackbar_undo_action.png "Pop-up message with undo action")
+
+#### [Toasts](https://developer.android.com/guide/topics/ui/notifiers/toasts.html)
+A toast provides simple feedback about an operation in a small popup. It only fills the amount of space required for the message and the current activity remains visible and interactive.
+
+![Toasts](https://developer.android.com/images/toast.png)
+
+There are many more UI elements available to your app than the few mentioned above. So, go explore the developer's page and the links listed under Resources.
+
+### Resources:
+Skim through the resources' content listed below. Whenever you start working on an app, you can come back to these and use the relevant information.
+
+  1. [Best Practices for User Interface](https://developer.android.com/training/best-ui.html)
+  2. [User Interface Guide](https://developer.android.com/guide/topics/ui/index.html)
+  3. [Material Design](https://material.google.com/)
+  4. [Android Design Principles](https://developer.android.com/design/get-started/principles.html)
+  5. [Material Design apps, icons, etc](https://material.uplabs.com/)
+
+### Try It Out!
+ 
+**[HW]** 
+  1. Download Facebook app on an Android phone (if you don't have access to either a Facebook account or an Android phone, let the teaching team know)
+  2. Observe and ask questions about the design intention. The design principles will help you here. Write up the questions and the corresponding findings?
+  3. You might have thoughts about a different layout. Prototype on paper. Include your prototype's picture (and its revisions) in your write-up. You could also use one of the programs (like Fluid UI) to make the prototypes.
+  4. (Optional) Recreate your layout(s) in Android Studio by writing XML file(s) for them.
+  
+Fill out [this survey](https://goo.gl/forms/LLUhDfZzBkkqawtp1) after you finish the HW
```

Comment:
Assuming they have to change their code, we should ask for a link to the repo where their code is (unless we just assume its in their class repo)